cmake/modules/BundleInstall.cmake.in

```diff
@@ -27,13 +27,17 @@ if(DEFINED APPLE_CODESIGN_IDENTITY AND DEFINED APPLE_CODESIGN_ENTITLEMENTS)
           "${PATH_TO_SIGN}"
       )
     else()
+      message(STATUS "Signing bundle with hardened runtime and identity ${APPLE_CODESIGN_IDENTITY}")
       execute_process(COMMAND
           codesign --verbose=4 --deep --force --options runtime
           --entitlements "${APPLE_CODESIGN_ENTITLEMENTS}"
           --sign "${APPLE_CODESIGN_IDENTITY}"
           "${PATH_TO_SIGN}"
       )
     endif()
+    if(NOT _EXIT_CODE EQUAL 0)
```

Comment:
what variable is this? I assume its the Cmake equivalent of `$?`? I can't find any docs on it.

Reply:
Exactly. I took it from some Stackoverflow post that I can't find anymore, but there seems to be a more elegant solution anyway: https://stackoverflow.com/a/72358749

Reply:
Oh actually, this was missing a `RESULT_VARIABLE _EXIT_CODE` to declare it, but I didn't notice because I was of course expecting it to fail: https://stackoverflow.com/a/65725647